cmds/exp/zbi/zbi.go

```diff
@@ -34,7 +34,7 @@ func main() {
 
 	imageJSON, err := json.MarshalIndent(image, "", "  ")
 	if err != nil {
-		log.Fatalf(err.Error())
+		log.Fatal(err.Error())
```

Comment:
as long as we're changing this can it just be log.Fatal? Same for the other ones.

Reply:
I am not sure what you mean by 

> can it just be log.Fatal?

 The Error function over a struct implementing the Error interface should always return a plain string. Also, the `log.Fatal` function takes plane strings as an argument and after printing does the same invocation of `os.Exit` as `log.Fatalf`.

Reply:
func Fatal(v ...[any])

we call log.Fatal with just err all over the place. You don't need err.Error()
log.Fatal(err) 
is just fine.